src/shafaque/Assignment16.java

```diff
@@ -0,0 +1,29 @@
+package shafaque;
+
+public class Assignment16 {
+	int sum= 0;
+	int count = 0;
+	void average(int startindex , int endindex)
+	{
+		System.out.println("Number are:" );
+		for(int num=startindex;num<= endindex;num++)
+		{
+			if(num%7 == 0 && num%5 == 0)
+			{
+			   System.out.println(num);
+				count++;
+				sum = sum + num;
+			}
+		}
+			System.out.println("sum is " +sum);
```

Comment:
this statements should be in same alignment with for loop and above print statement